app/policies/photo_policy.rb

```diff
@@ -17,27 +14,15 @@ def create?
     true
   end
 
-  def new?
-    create?
-  end
-
   def update?
     user == photo.owner
   end
```

Comment:
You might want to revise it: currently, when a user is trying to update their own photo, they are not authorized for that